variables.tf

```diff
@@ -118,6 +118,24 @@ variable "version_stages" {
   default     = null
 }
 
+variable "create_random_password" {
+  description = "Determines whether a random password will be generated"
+  type        = bool
+  default     = false
+}
+
+variable "random_password_length" {
+  description = "The length of the generated random password"
+  type        = number
+  default     = 32
+}
+
+variable "random_password_override_special" {
+  description = "Supply your own list of special characters to use for string generation. This overrides the default character list in the special argument"
+  type        = string
+  default     = "!#$%&*()-_=+[]{}<>:?"
```

Comment:
Do you know where these chars came from? There are no `/.,\~`, for example. Is there some kind of document containing such a set?

Reply:
whoops - looks like I copied the example value and not the actual default value. Those are coming from here https://registry.terraform.io/providers/hashicorp/random/latest/docs/resources/password#special

I believe the others you listed tend to cause issues in (I know some of those are not supported in RDS secrets depending on the DB engine)

Reply:
also it looks like I don't have permission on this repo yet if you wouldn't mind adding/merging when you get a chance

Reply:
I've just fixed permissions and merged.